src/index.ts

```diff
@@ -1,16 +1,27 @@
 import path from "path";
 import cors from 'cors';
-import express from 'express';
+import express, { Request } from 'express';
 import { getImage, getDomain } from './domain';
 require('dotenv').config()
 
+interface RequestParams {
+  tokenId?: string;
+}
+
 const app = express();
 
 app.get('/', (req, res) => {
   res.send('Well done mate!');
 })
 
-app.get('/name/:tokenId', async function (req, res) {
+app.get('/name/:tokenId?', async function (req: Request<RequestParams>, res, next) {
+  if(!req.params.tokenId) {
+    res.status(404).json({
```

Comment:
We should also return a 404 if a tokenid is provided but that token doesn't exist.

Reply:
Replied here https://github.com/ensdomains/ens-metadata-service/pull/2#discussion_r664117588